beacon-chain/blockchain/receive_block_test.go

```diff
@@ -137,12 +145,11 @@ func TestService_ReceiveBlock(t *testing.T) {
 			require.NoError(t, err)
 			err = s.ReceiveBlock(ctx, wsb, root)
 			if tt.wantedErr != "" {
-				assert.ErrorContains(t, tt.wantedErr, err)
+				require.ErrorContains(t, tt.wantedErr, err)
```

Comment:
This one can be left as an assert

Reply:
it should just be a require, as the test now fails here